test/unit/convertV2.test.js

```diff
@@ -1107,6 +1108,25 @@ describe('The convert v2 Function', function() {
       });
   });
 
+  it('Should convert xml request body with complete string example correctly', function(done) {
+    const openapi = fs.readFileSync(xmlrequestExampleBody, 'utf8');
+    Converter.convert({ type: 'string', data: openapi },
```

Comment:
If this is a `convertV2` test, shouldn't we use the `convertV2` method? And in that case `requestParametersResolution` will not be applicable right?

Reply:
Yes, missed this. Have updated test to use correct option and interface.